### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_FilesTab_Keyboard:Keyboard focus is not visible on the "Upload" button.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#3954]
**EnvironmentDetails:**
Application Name:AzureNotebooks_Jyupter
Login Url: notebooks.azure.com
BrowserDetails:
Edge:Microsoft Edge 41.16299.611.0
Chrome:Version 68.0.3440.106 
Operating System:Windows10
 
 **Steps to reproduce:**
1.Login with the appropriate credentials to azure notebook account.    
2.Go to Get started à libraries à Select any of the libraries à click on clone to clone the library
3.Click on Run or right click and select Run to run the notebook.
5.Navigate to the "Upload" link using the "Tab" key.
 **Actual:**
Keyboard focus is not visible on the "Upload" button.
**Expected:**
Keyboard focus should be visible on the "Upload" button.
**UserImpact:**
User will face difficulty while traversing through the page. 
**Attachment:**
[A11y_AzureNotebooks_Jupyter_FilesTab_Keyboard1.pptx](https://github.com/jupyter/notebook/files/2399882/A11y_AzureNotebooks_Jupyter_FilesTab_Keyboard1.pptx)

**Comment by fabihaahmed** [comment]
This issue solved by the issue #4009.

**Comment by jtpio** [comment]
Closing as there are now different issues tracking accessibility improvements for Jupyter Notebook 7, for example https://github.com/jupyter/notebook/issues/6800

Thanks!
